pandas/util/testing.py

```diff
@@ -1419,6 +1419,31 @@ def assert_copy(iter1, iter2, **eql_kwargs):
                                     % (type(elem1), type(elem2)))
 
 
+def assert_numpy_arrays_copy(iter1, iter2):
+    """
```

Comment:
no, don't add these we are only calling them once. These need to very specifiy test in the Index section where you use them. (define them as check functions local to the test case).